pkg/utils/urls/replace.go

```diff
@@ -21,5 +21,25 @@ func ReplaceHost(orgURL, host string) string {
 		u.Scheme = ""
 	}
 	u.Host = host
-	return strings.TrimPrefix(u.String(), "//")
+	return strings.ReplaceAll(strings.TrimPrefix(u.String(), "//"), "%2F", "/")
+}
+
+// ToAPIEndpoint turns URL to a valid API endpoint used in
```

Comment:
I think we need more detail on the allowed formats of `url` beacuse, as is, I don't think I'd know how or what this function can be used for. I'm also not sure it belongs in the `urls` package because its focused heavily on registries rather than generic URL manipulation.